integration/spec/features/v2/save_and_return_spec.rb

```diff
@@ -39,10 +41,43 @@
     check_optional_text(page.text)
     form.email_confirmation.set('email@email.com')
     check_validation_error_message("Check your email address is correct")
-    form.email_confirmation.set('email@example.com')
+    form.email_confirmation.set('fb-acceptance-tests@digital.justice.gov.uk')
     continue
     check_optional_text(page.text)
     expect(page.text).to include('Your form has been saved')
-    expect(page.text).to include('We have sent a one-off link to email@example.com')
+    expect(page.text).to include('We have sent a one-off link to fb-acceptance-tests@digital.justice.gov.uk')
+
+    resume_progress_email = get_resume_email('save-and-return-v2-acceptance-test')
+    resume_link = extract_link_from_email(resume_progress_email)
+    
+    visit resume_link
+
+    expect(page.text).to include('Continue with "save-and-return-v2-acceptance-test"')
+    form.resume_secret_answer.set('bar')
+    continue
+    check_validation_error_message('Your answer is incorrect. You have 3 attempts remaining.')
+    form.resume_secret_answer.set('foo')
+    continue
+    # expect(page.text).to include('You have sucessfuly retrieved your saved information.')
```

Comment:
can we remove this comment please?

Reply:
Now updated, was there because I'd missed that content in the actual page :D